runtests.py

```diff
@@ -5,49 +5,57 @@
 #
 
 """
-File for running tests programmatically.
+Script for running Spyder tests programmatically.
 """
 
 # Standard library imports
 import os
 import sys
+import argparse
 
 # Third party imports
-import qtpy  # to ensure that Qt4 uses API v2
 import pytest
 
 
 # To run our slow tests only in our CIs
-run_slow = False
-if os.environ.get('CI', None) is not None or '--run-slow' in sys.argv:
-    run_slow = True
+RUN_CI = os.environ.get('CI', None) is not None
+run_slow = RUN_CI
```

Comment:
This is very confusing and I don't like it. `run_slow` is not the same as `RUN_CI` (which is what you're equating here), so please find a better way to write this or just leave them separate:

    RUN_CI = os.environ.get('CI', None) is not None
    RUN_SLOW = False

Reply:
Okay, however you prefer. I just dropped ``run_slow`` entirely then since it doesn't actually do anything (different from ``RUN_CI``) in ``3.x``; I did it that way originally to make your life a little easier when merging it. When you merge, you'll need to do something about it so the ``run_slow`` tests skip on Darwin, but that's up to you ofc.